torch/jit/_shape_functions.py

```diff
@@ -426,6 +426,14 @@ def squeeze(li: List[int], dim: int):
             out.append(li[i])
     return out
 
+def squeeze_dims(li: List[int], dims: List[int]):
+    wrapped_dims : List[int] = []
+    for i in dims:
+        wrapped_dims.append(maybe_wrap_dim(i, len(li)))
+    wrapped_dims.sort(reverse=True)
+    for i in wrapped_dims:
+        li = squeeze(li, i)
+    return li
```

Comment:
do you need add_shape_compute_mapping?

(also, can you comment in the PR clarifying that the _native_batch_norm_legit and upsample_nearest2d changes are unrelated & only occur because gen_jit_shape_functions.py wasn't run when the changes affecting the codegen changed?)